src/main/java/ua/rent/masters/easystay/service/impl/TelegramNotificationService.java

```diff
@@ -0,0 +1,37 @@
+package ua.rent.masters.easystay.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ua.rent.masters.easystay.dto.NotifyResponse;
+import ua.rent.masters.easystay.handler.TelegramBotHandler;
+import ua.rent.masters.easystay.model.Accommodation;
+import ua.rent.masters.easystay.model.Booking;
+import ua.rent.masters.easystay.model.Payment;
+import ua.rent.masters.easystay.model.User;
+import ua.rent.masters.easystay.service.NotificationService;
+
+@Service
+@RequiredArgsConstructor
+public class TelegramNotificationService implements NotificationService {
+    private final TelegramBotHandler botHandler;
+
+    @Override
+    public void notifyAboutAccommodation(Accommodation accommodation, User user) {
```

Comment:
A bit difficult to understand. Notify about what? Creation, change or something else

Reply:
It is better?
```suggestion
    public void notifyAboutAccommodationChanges(Accommodation accommodation, User user, ChangeStatus) {
```